tests/api_tests/test_participant_api.py

```diff
@@ -845,6 +848,23 @@ def test_participant_post_skips_summary_update(self, summary_update_mock):
         })
         summary_update_mock.assert_not_called()
 
+    def test_bypass_origin_check_in_dev_config(self):
+        response = self.send_post("Participant", self.participant)
+        participant_id = response["participantId"]
+        new_user_info = deepcopy(config.getSettingJson(config.USER_INFO))
+        if new_user_info["example@cebyok.com"]["bypassOriginCheck"]:
+            get_response = self.send_get("Participant/%s" % participant_id)
+        self.assertEqual(response, get_response)
```

Comment:
Dot think this is working how you expect? Not seeing how the cybok user is evaluated in thsi GET request?
This defaults to the example@example.com user.